docs/how-to-create-model.md

```diff
@@ -125,6 +125,35 @@ int main(int argc, char *argv[]) {
 ```
 
 ## Build and Run "MyApp" application
+
+
+### Add  meson.build file under `MyApp/jni`
+
+In order to build MyApp application, it is required to add `meson.build` for it. To this end, create the build file under `Applications/MyApp/jni/`.
+Here is an example of `meson.build` file of `MyApp` application.
+
+```
+myapp_sources= [
+    'main.cpp',
+    cifar_path / 'cifar_dataloader.cpp'
+]
+
+myapp_dependencies = [app_utils_dep,
+  nntrainer_dep,
+  nntrainer_ccapi_dep
```

Comment:
```nntrainer_ccapi_dep``` includes ```nnstrainer_dep```. You don't need to add ```nntrainer_dep``` here: https://github.com/nnstreamer/nntrainer/blob/main/api/ccapi/meson.build

Reply:
I totally missed it. Thank you for pointing it out.